<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SceneVizWidget</class>
 <widget class="QWidget" name="SceneVizWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>476</width>
    <height>418</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SceneVizWidget</string>
  </property>
  <widget class="QPushButton" name="toggleRenderWindowButton">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>5</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Hide Window</string>
   </property>
  </widget>
  <widget class="QFrame" name="dockFrame">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>94</y>
     <width>461</width>
     <height>321</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QDockWidget" name="dockWidget">
    <property name="geometry">
     <rect>
      <x>1</x>
      <y>0</y>
      <width>459</width>
      <height>317</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="features">
     <set>QDockWidget::AllDockWidgetFeatures</set>
    </property>
    <widget class="QWidget" name="dockWidgetContents">
     <property name="cursor">
      <cursorShape>ArrowCursor</cursorShape>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QGroupBox" name="birdsGroupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>5</y>
     <width>120</width>
     <height>80</height>
    </rect>
   </property>
   <property name="title">
    <string>Birds</string>
   </property>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>121</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <property name="leftMargin">
      <number>4</number>
     </property>
     <property name="topMargin">
      <number>4</number>
     </property>
     <property name="rightMargin">
      <number>4</number>
     </property>
     <property name="bottomMargin">
      <number>4</number>
     </property>
     <property name="verticalSpacing">
      <number>6</number>
     </property>
     <item row="0" column="0">
      <widget class="QCheckBox" name="baseCheckBox">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Base</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QCheckBox" name="tipCheckBox">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Tip</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QCheckBox" name="instrCheckBox">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Instr</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QCheckBox" name="chestCheckBox">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Chest</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="ultrasoundGroupBox">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>5</y>
     <width>91</width>
     <height>80</height>
    </rect>
   </property>
   <property name="title">
    <string>Ultrasound</string>
   </property>
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>94</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QCheckBox" name="imgPlaneCheckBox">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Imaging Plane</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>5</y>
     <width>141</width>
     <height>80</height>
    </rect>
   </property>
   <property name="title">
    <string>Models</string>
   </property>
   <widget class="QRadioButton" name="noTformRadio">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>84</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>No Tform</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="tform1Radio">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>84</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Tform Opt1</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="tform2Radio">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>84</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Tform Opt2</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="usAngleSpinBox">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>20</y>
      <width>43</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>30</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Recenter</string>
   </property>
  </widget>
  <widget class="QPushButton" name="resetBaseButton">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>60</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Reset Base</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
